src/packages/@app/components/App.svelte

```diff
@@ -70,6 +71,9 @@
     user.subscribe((value) => {
       isloggedIn = value;
     });
+    // Connect WebSocket
+    const wsService = new AiAssistantWebSocketService();
+    wsService.connectWebSocket();
 
```

Comment:
we can initiate this code in dashboard file

Reply:
done